releases/5.1.0.md

```diff
@@ -0,0 +1,182 @@
+# New minor release! Turf 5.1.0 🎉
+
+TurfJS releases are being tracked using [Milestones](https://github.com/Turfjs/turf/milestone/13?closed=1).
+
+## Contributors
+
+- @muziejus 🥇
+- @rowanwins 🥇
+- @DenisCarriere
+- @tmcw
+- @stebogit
+- @woutervh-
+- @dwaggenspack
+
+### Crowdsourced Funding
+
+#### Why we're looking for support
+
+TurfJS is a community-driven project maintained by a small group of core contributors who work on the project in their spare time. Time is spent supporting users, improving documentation, fixing bugs as well as creating new modules. Your funding will directly go to development costs, marketing campaigns, promotional events & any other financial costs to operate & maintain TurfJS.
+
+In particular, we're looking for corporate sponsors who use TurfJS in revenue-generating ways, either by creating applications for clients, or through use in an app used by customers. Of course individuals are welcome to support us as well if TurfJS has helped you :)
+
+https://opencollective.com/turf
+
+Help TurfJS by becoming a **Backer** for only 2$/mo.
+
+## 🚀 New Modules
+
+### [`@turf/center-median`][center-median]
+
+Takes a FeatureCollection of points and calculates the median center, algorithimically. The median center is understood as the point that is requires the least total travel from all other points.
+
+(PR https://github.com/Turfjs/turf/pull/1122 - Author @muziejus)
+
+
+### [`@turf/standard-deviational-ellipse`][standard-deviational-ellipse]
+
+Takes a FeatureCollection and returns a standard deviational ellipse, also known as a “directional distribution.” The standard deviational ellipse aims to show the direction and the distribution of a dataset by drawing an ellipse that contains about one standard deviation’s worth (~ 70%) of the data.
+
+This module mirrors the functionality of Directional Distribution in ArcGIS and the QGIS Standard Deviational Ellipse Plugin
+
+(PR https://github.com/Turfjs/turf/pull/1120 - Author @muziejus)
+
+
+### [`@turf/center-mean`][center-mean]
+
+Takes a Feature or FeatureCollection and returns the mean center. Can be weighted.
+
+(PR https://github.com/Turfjs/turf/pull/1089 - Author @muziejus)
+
+
+### [`@turf/ellipse`][ellipse]
+
+Takes a Point and calculates the ellipse polygon given two semi-axes expressed in variable units and steps for precision.
+
+(PR https://github.com/Turfjs/turf/pull/1087 - Author @muziejus)
+
+## 📖 Documentation
+
+- TurfJS's website has a fresh new look! 🚀 http://turfjs.org (Author @rowanwins PR https://github.com/Turfjs/turf-www/pull/127)
+  - This uses nuxt to generate the pages as static html which should be better for search engines
+  - The docs are now all on a single page
+  - Built sizes are much better than the current
+  - The map moves to a module when it's description comes into view, I think this functions reasonably nicely.
```

Comment:
"when its description"

Reply:
When you scroll down to the specific module, then the map appears, instead of loading every single map (that was the previous behavior)

Reply:
I just meant that the apostrophe isn't needed 😆

Reply:
Oh :) lol